wordpress-mobile/WordPress-Android
# Adding user profile bottom sheet

Part of the #14137 and #14138 

This PR adds a user profile bottom sheet to the list of likers for both notifications and reader. Companion Fluxc PR [here](https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/1947)

NOTE: it requires D59220 endpoint changes to be shipped before it can be merged (now deployed). Note also that to check the CI built APK you need to deploy the above mentioned differential to your sandbox until those endpoint changes are not shipped.

<p align=center><image src="https://user-images.githubusercontent.com/47797566/114324279-8444d500-9b29-11eb-96fb-71cc2d29e577.png" width=300/></p>



### To test

Design has already been verified and approved. Should be enough to go to list of likers (both from reader or notifications) and open the user profile bottom sheet from there for users listed. 

Check that:
- the information presented are consistent with those of the user tapped on.
- when bio is missing you the relevant section is not shown
- when no preferred blog is present, all the Site section is not shown

During testing, the site name was not decoded correctly so that for example things like "It's great" should have resulted in something like "it&#039;s great". In our understanding now the text fields on the bottom sheet should be encoded/decoded correctly but good to have some attention for that, thanks 🙇 .


## Regression Notes
1. Potential unintended areas of impact
Feature still behind feature flag.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Feature still behind feature flag.

3. What automated tests I added (or what prevented me from doing so)
Feature still behind feature flag.

PR submission checklist:
